plugins/chat/assets/javascripts/discourse/routes/chat-channel-thread.js

```diff
@@ -0,0 +1,21 @@
+import DiscourseRoute from "discourse/routes/discourse";
+import { inject as service } from "@ember/service";
+
+export default class ChatChannelThread extends DiscourseRoute {
+  @service router;
+  @service chatThreadsManager;
+  @service chatStateManager;
+  @service chat;
+
+  async model(params) {
+    return this.chatThreadsManager.find(
+      this.chat.activeChannel.id,
+      params.threadId
+    );
```

Comment:
```suggestion
    return this.chatThreadsManager.find(
      modelFor("chat.channel").id,
      params.threadId
    );
```

I prefer to use the URL as the source of truth

Reply:
Ah yes good point, thanks :+1: